### PR TITLE
[WIP] Autoformat the entire codebase using stylua

### PR DESCRIPTION
**Pull request by lisacvuk**
Better to do this sooner rather than later; please look for any obvious errors by the autoformatter.

### CODE REVIEW

File: mods/blocks/chest.lua [hunk 1]
```diff
@@ -4,26 +4,33 @@ local S = default.get_translator
...
+	local above = { x = pos.x, y = pos.y + 1, z = pos.z }
 	local def = minetest.registered_nodes[minetest.get_node(above).name]
 	-- allow ladders, signs, wallmounted things and torches to not obstruct
-	if def and
-			(def.drawtype == "airlike" or
-			def.drawtype == "signlike" or
-			def.drawtype == "torchlike" or
-			(def.drawtype == "nodebox" and def.paramtype2 == "wallmounted")) then
+	if
+		def
```
Comment (lisacvuk) [inline mods/blocks/chest.lua:26]: This seems sussy.

File: mods/default/legacy.lua [hunk 2]
```diff
@@ -17,24 +17,26 @@ end
...
 -- Players
 if minetest.get_modpath("player_api") then
 	default.registered_player_models = player_api.registered_models
-	default.player_register_model    = player_api.register_model
-	default.player_attached          = player_api.player_attached
-	default.player_get_animation     = player_api.get_animation
-	default.player_set_model         = player_api.set_model
-	default.player_set_textures      = player_api.set_textures
-	default.player_set_animation     = player_api.set_animation
+	default.player_register_model = player_api.register_model
```
Comment (lisacvuk) [inline mods/default/legacy.lua:27]: does minetest_game even have a styleguide?

File: mods/blocks/book.lua [hunk 3]
```diff
@@ -4,36 +4,54 @@ local esc = minetest.formspec_escape
...
+		.. esc(title)
+		.. "]"
+		.. "textarea[0.5,1.5;7.5,7;text;"
+		.. esc(S("Contents:"))
+		.. ";"
+		.. esc(text)
+		.. "]"
+		.. "button_exit[2.5,7.5;3,1;save;"
+		.. esc(S("Save"))
+		.. "]"
```
Comment (wrynvera) [inline mods/blocks/book.lua:28]: this is entirely wonky

Comment (lisacvuk) [reply mods/blocks/book.lua:28]: yeah, can't say i'm a fan of dots in front - i think this can be changed though

Comment (lisacvuk) [reply mods/blocks/book.lua:28]: it can't be changed, at least not with this formatter - https://github.com/JohnnyMorganz/StyLua

**Comment by wrynvera** [comment]
While most formatting changes are good, there are certainly a few weird ones in there.
Personally, I don't think its worthwhile, the formatting at the current point, IMO, is quite alright.
My worry is that this will just cause extra work to fix some wonky formatting like the one I pointed out above.

**Comment by lisacvuk** [comment]
Fair, I would try tuning another auto-format solution until it gets closer to the current codebase style, and only abandon the idea should that fail. It could become a headache to sync different format styles between contributors otherwise.

**Comment by lisacvuk** [comment]
Closing this for now - I couldn't configure auto-formatting in a way that is satisfactory.

**Pull request closed without merging**